Regression/PrimevalTitmouse/Animations.cs

```diff
@@ -515,11 +493,73 @@ public static void Say(string[] msgs, Body b = null)
             Animations.Say(Strings.RandString(msgs), b);
         }
 
-        public static Microsoft.Xna.Framework.Rectangle UnderwearRectangle(Container c, string type = null, int height = LARGE_SPRITE_DIM)
+        public static Microsoft.Xna.Framework.Rectangle UnderwearRectangle(Container c, FullnessType type = FullnessType.None, int height = LARGE_SPRITE_DIM)
         {
             if (c.spriteIndex == -1)
                 throw new Exception("Invalid sprite index.");
-            int num = type != null ? (!(type == "drying") ? (!(type == "messy") ? (!(type == "wet") ? 0 : LARGE_SPRITE_DIM) : LARGE_SPRITE_DIM*2) : LARGE_SPRITE_DIM*3) : (!c.IsDrying() ? ((double)c.messiness <= 0.0 ? ((double)c.wetness <= 0.0 ? 0 : LARGE_SPRITE_DIM) : LARGE_SPRITE_DIM*2) : LARGE_SPRITE_DIM*3);
+            int num = 0;
+            //Using switch statement instead of ternary operator for better readability and to add another type.

```

Comment:
Good idea. Should delete commented out code below.

Reply:
Done